ibis/backends/tests/test_array.py

```diff
@@ -711,10 +722,28 @@ def test_array_union(con):
 @pytest.mark.notimpl(
     ["sqlite"], raises=NotImplementedError, reason="Unsupported type: Array..."
 )
-def test_array_intersect(con):
-    t = ibis.memtable(
-        {"a": [[3, 2], [], []], "b": [[1, 3], [None], [5]], "c": range(3)}
-    )
+@pytest.mark.parametrize(
+    "data",
+    [
+        param(
+            {"a": [[3, 2], [], []], "b": [[1, 3], [None], [5]], "c": range(3)},
+            id="nulls",
+            marks=[
+                pytest.mark.notyet(
+                    ["bigquery"],
+                    raises=BadRequest,
+                    reason="BigQuery doesn't support arrays with null elements",
```

Comment:
Previously these tests passed because of BigQuery's quirky UNNEST behavior

1. We were creating data with `UNNEST(<ARRAY<...>>[...])`
2. The `ArrayIntersect` operation is implemented using `SELECT ... UNNEST(left) INTERSECT DISTINCT SELECT UNNEST(right)`

The difference comes from this quirk:

```sql
SELECT [1, NULL]; -- NOT allowed
SELECT * FROM UNNEST([1, NULL]); -- allowed
```

Reply:
No, I don't think this warrants a breaking change commit.